<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaCadastroClientes</class>
 <widget class="QDialog" name="JanelaCadastroClientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente</string>
  </property>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>880</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clientes cadastrados:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Complemento:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_complementoCliente"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pb_ave">
        <property name="text">
         <string>Ave</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_gato">
        <property name="text">
         <string>Gato</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pb_cachorro">
        <property name="text">
         <string>Cachorro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_tartaruga">
        <property name="text">
         <string>Tartaruga</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar Cliente:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereço:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Peso*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_pesoAnimal"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Kg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>104</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Idade*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_idadeCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Nome*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_nomeCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cbox_animaisCadastrados">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>850</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>CPF*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_cpfCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>790</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*Campo obrigatório</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_nomeAnimal"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>950</y>
     <width>447</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QPushButton" name="pb_consultarCliente">
      <property name="text">
       <string>Consultar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_alterarCliente">
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_removerCliente">
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pb_inserirCliente">
      <property name="text">
       <string>Inserir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_limparCliente">
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Rua:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_ruaCliente"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Bairro*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_bairroCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar Animal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>850</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Animais cadastrados:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_clientesCadastrados">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>880</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>760</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Cidade*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_cidadeCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>88</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Nº*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_numeroCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>104</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Idade*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_idadeAnimal"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Estado*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbox_estadoCliente">
      <item>
       <property name="text">
        <string>- - -</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RJ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TO</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_outroAnimal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Outro Animal</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>157</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sexo*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_macho">
      <property name="text">
       <string>Macho</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_femea">
      <property name="text">
       <string>Femea</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>520</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>CEP*:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_cepCliente">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
